app/templates/main/browse-consignment.html

```diff
@@ -99,340 +71,247 @@
-                                   name="width10"
-                                   type="text">
+                        <div class="govuk-form-group govuk-form-group--consignment-filter">
+                            <input class="govuk-input govuk-!-width-full govuk-input--consignment-input"
+                                   id="full"
+                                   name="file_type"
+                                   type="text"
+                                   value="{% if user_filters['file_type'] %}{{ user_filters['file_type'] }}{% endif %}">
                         </div>
                     </div>
-                    <div class="filters-form__date__container">
-                        <div class="govuk-form-group date-form filters-form__group filters-form__date-from-group">
-                            <p class="govuk-body date-form browse__body filters-form__body-date">Date from</p>
-                            <fieldset class="govuk-fieldset" role="group" aria-describedby="date-from">
-                                <div class="govuk-date-input" id="date-from">
-                                    <div class="govuk-date-input__item">
-                                        <div class="govuk-form-group">
-                                            <input class="govuk-input govuk-date-input__input govuk-input--width-2"
-                                                   id="date-from-day"
-                                                   name="date-from-day"
-                                                   type="text"
-                                                   inputmode="numeric">
-                                            <label class="govuk-label govuk-date-input__label filters-form__date__label"
-                                                   for="date-from-day">(DD)</label>
+                    <!-- DATES -->
+                    <div class="consignment-filter-container">
+                        <h3 class="govuk-heading-s govuk-heading-s--consignment-filter">Dates</h3>
+                        <div class="govuk-form-group govuk-form-group__consignment-form-group">
+                            <fieldset class="govuk-fieldset">
+                                <div class="govuk-radios govuk-radios--small" data-module="govuk-radios">
+                                    <div class="govuk-radios__item">
+                                        <input class="govuk-radios__input" id="date_last_modified" name="date_filter_field"
+                                            type="radio" value="date_last_modified" {% if
+                                            user_filters["date_filter_field"]=='date_last_modified' %}checked{% endif %}>
+                                            <label class="govuk-label govuk-radios__label govuk-radios__label--consignment"
+                                                   for="date_last_modified">File last modified</label>
```

Comment:
I think we have agreed on Date last modified being called Record date in the frontend

Reply:
Colin is already working on this ticket and will pick up tomorrow my latest changes on consignment view

Reply:
as in @colinbowen is working on https://national-archives.atlassian.net/browse/AYR-718 ?

Reply:
Will be picked up as part of AYR-718 & AYR-714